bigbite/Jake-Week-2
# Feat/resources page

## Description
Created a resources page for a Mental Health Awareness website that includes; a navigation bar, introductory image title and paragraph, 3 dropdown boxes that upon click show a description, a list (without ul or li) of information sources (each containing an image, title, paragraph, and appropriate link), 3 treatment information boxes(each containing an image, title, paragraph, and appropriate link), and a footer.


## Change Log



- Added an introductory section, utilising backdrop-filter for the first time.
- Added 3 dropdown boxes that each upon click reveal a description.
- Added 3 information containers displayed in a list type of style each containing appropriate links.
- Added 3 information containers utilising display-flex to display them horizontally. Each contains an appropriate link as well as having hover functionality.
- Imported navbar from previous page (and added link).
- Imported footer.
- Added some new css variables.
- Experimented with @keyframes for the first time.

## Steps to test
For the dropdown boxes section, open dev tools and ensure that the class of 'description-content-show' and 'description-content-hide' is added/removed to the container that has been pressed. The function used utilises id's and an if else statement to ensure it is targeting the correct container.
For the hover functionality, ensure that when a container is hovered it scales up slightly.


## Screenshots/Videos


Dropdown:

https://github.com/bigbite/Jake-Week-2/assets/53060167/7b63b251-2889-41f3-a9f4-0bf9fd034de4

Hover functionality:

https://github.com/bigbite/Jake-Week-2/assets/53060167/84316361-3310-45ca-b70e-0e604686bf88


## Checklist:
